src/main/webapp/app/entities/club-family/family-member-create.tsx

```diff
@@ -110,7 +110,7 @@ export class FamilyMemberCreate extends React.Component<IFamilyMemberCreateProps
         <Row className="justify-content-center">
           <Col md="8">
             <h2>
-              <Translate contentKey="clubmanagementApp.clubFamily.member.createLabel">Add Family Member</Translate>
+              <Translate contentKey="clubmanagementApp.clubFamily.member.createOrEditLabel">Add or edit a CC Family Member</Translate>
```

Comment:
I mean is like this
![image](https://user-images.githubusercontent.com/35193728/136658592-2d82c86b-bada-4302-a008-ddf75feb0e32.png)